### PR TITLE
Update release plan

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
Update plan for the upcoming releases after v1.42

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: docs/development/process.md [hunk 1]
```diff
@@ -16,15 +16,20 @@ Hotfixes are usually maintained for the latest three minor releases, though, the
...
-v1.39   | Week 01-02  | January 3, 2022        | January 16, 2022   | [@ialidzhikov](https://github.com/ialidzhikov)     |
-v1.39   | Week 03-04  | January 17, 2022       | January 30, 2022   | [@timuthy](https://github.com/timuthy)             |
-v1.40   | Week 05-06  | January 31, 2022       | February 13, 2022  | [@BeckerMax](https://github.com/BeckerMax)         |
-v1.41   | Week 06-07  | February 14, 2022      | February 27, 2022  | [@stoyanr](https://github.com/stoyanr)             |
-v1.42   | Week 08-09  | February 28, 2022      | March 13, 2022     | [@voelzmo](https://github.com/voelzmo)             |
+v1.39   | Week 01-04  | January 3, 2022        | January 30, 2022   | [@ialidzhikov](https://github.com/ialidzhikov), [@timuthy](https://github.com/timuthy) |
+v1.40   | Week 05-06  | January 31, 2022       | February 13, 2022  | [@BeckerMax](https://github.com/BeckerMax)                                             |
+v1.41   | Week 06-07  | February 14, 2022      | February 27, 2022  | [@plkokanov](https://github.com/plkokanov)                                             |
```
Comment (timebertt) [inline docs/development/process.md]:
```suggestion
v1.40   | Week 05-06  | January 31, 2022       | February 13, 2022  | [@BeckerMax](https://github.com/BeckerMax)                                             |
v1.41   | Week 07-08  | February 14, 2022      | February 27, 2022  | [@plkokanov](https://github.com/plkokanov)                                             |
```
same for the following weeks (numbers are shifted by 1)

**Review by timebertt — COMMENTED** [review]
Thanks!
Ignoring the misaligned week numbers, this schedule works for me.

**Review by timebertt — DISMISSED** [review]
/lgtm

**Review by timuthy — DISMISSED** [review]
/lgtm

**Review by ialidzhikov — DISMISSED** [review]
/lgtm

File: docs/development/process.md [hunk 2]
```diff
@@ -16,15 +16,20 @@ Hotfixes are usually maintained for the latest three minor releases, though, the
...
+v1.45   | Week 15-16  | April 11, 2022         | April 24, 2022     | [@danielfoehrKn](https://github.com/danielfoehrKn)                                     |
+v1.46   | Week 17-18  | April 25, 2022         | May 8, 2022        | [@ialidzhikov](https://github.com/ialidzhikov)                                         |
+v1.47   | Week 19-20  | May 9, 2022            | May 22, 2022       | [@stoyanr](https://github.com/stoyanr)                                                 |
+v1.48   | Week 21-22  | May 23, 2022           | June 5, 2022       | [@rfranzke](https://github.com/rfranzke)                                               |
+v1.49   | Week 23-24  | June 6, 2022           | June 19, 2022      | [@timuthy](https://github.com/timuthy)                                                 |
+v1.50   | Week 25-26  | June 20, 2022          | July 3, 2022       | [@shafeeqes](https://github.com/shafeeqes)                                             |
+v1.51   | Week 27-28  | July 4, 2022           | June 17, 2022      | [@acumino](https://github.com/acumino)                                                 |
```
Comment (shafeeqes) [inline docs/development/process.md]:
```suggestion
v1.51   | Week 27-28  | July 4, 2022           | July 17, 2022      | [@acumino](https://github.com/acumino)                                                 |
```

**Review by timebertt — APPROVED** [review]

**Pull request merged**